apps/web/src/hooks/useCatchTxError.tsx

```diff
@@ -117,15 +120,7 @@ export default function useCatchTxError(params?: Params): CatchTxErrorReturn {
 
       return null
     },
-    [
-      throwUserRejectError,
-      toastSuccess,
-      t,
-      waitForTransaction,
-      handleNormalError,
-      handleTxError,
-      params?.waitForTransactionTimeout,
-    ],
+    [toastSuccess, t, waitForTransaction, params, throwUserRejectError, handleNormalError, handleTxError],
```

Comment:
the `params` will probably be a new instance on every render? Would be better if we list `params?.throwCustomeError` explicitly?